meinberlin/assets/scss/components/_button.scss

```diff
@@ -146,7 +146,7 @@
 .btn--full {
     display: block;
     width: 100%;
-    margin-bottom: $r-spacer;
+    // margin-bottom: $r-spacer;
```

Comment:
Did you check, what it does to the other uses of the wide button? If so, can this line be deleted?

Reply:
only in component library but no buttons should have a margin, i'm not sure why this one does?! I thought testing as we go with that one

Reply:
Oh, sorry, cache!